WordPress/src/main/java/org/wordpress/android/models/Capability.java

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.models;
+
+/**
+ * Used to decide what can the current user do in a particular blog
+ * A list of capabilities can be found in: https://codex.wordpress.org/Roles_and_Capabilities#Capabilities
+ */
+public enum Capability {
+    EDIT_USERS("edit_users"), // Check if user can change another user's role
+    LIST_USERS("list_users"), // Check if user can visit People page
+    PROMOTE_USERS("promote_users"); // Check if user can invite another user
+
+    private final String label;
+
+    Capability(String label) {
+        this.label = label;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Not an error but I would suggest adding a getter for `label` instead of overriding the enum's `toString` for this purpose. It would be more readable (to the callers of the getter) and wouldn't hijack the more generally purposed `toString()`. Cheers!